### PR TITLE
Tag Blink v0.4.0 [https://github.com/JunoLab/Blink.jl]

### PR DESCRIPTION
**Pull request by spencerlyon2**
Ref https://github.com/JunoLab/Blink.jl/issues/63

### CODE REVIEW

**Comment by tkelman** [comment]
can you push the git tag to https://github.com/JunoLab/Blink.jl/releases ?

**Comment by spencerlyon2** [comment]
done!



// Spencer

On August 11, 2016 at 1:21:36 PM, Tony Kelman (notifications@github.com)
wrote:

can you push the git tag to https://github.com/JunoLab/Blink.jl/releases ?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/JuliaLang/METADATA.jl/pull/5915#issuecomment-239145397>,
or mute the thread

**Comment by tkelman** [comment]
Compat is being used, needs to be listed explicitly in REQUIRE (with a minimum version that implements the needed features)

**Comment by spencerlyon2** [comment]
Ok I added to require, updated tag, updated metadata pr and pushed here.

We should be good to go

File: Blink/versions/0.4.0/requires [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+HttpServer
+Lazy
+JSON
+Requires
+Reexport
+Mustache
+MacroTools
+Mux
+BinDeps
+Compat 0.7.15
```
Comment (tkelman) [inline Blink/versions/0.4.0/requires]: this isn't new enough for `if is_apple()`

https://github.com/JuliaLang/Compat.jl/commit/85f886227c332507ec5b31627e0b3a9fbb6aa274

**Comment by spencerlyon2** [comment]
Ok I realized that the master branch doesn't work on Julia 0.4 -- so I backed up and tagged a new 0.4 compatible release along the Blink 0.3 minor line. Then I updated require for compat and Julia versions for master to work on 0.5

I think this should be good to go.

File: Blink/versions/0.4.0/requires [hunk 2]
```diff
@@ -0,0 +1,12 @@
+julia 0.5
```
Comment (tkelman) [inline Blink/versions/0.4.0/requires]: this won't be installable until the final release

Comment (spencerlyon2) [inline Blink/versions/0.4.0/requires]: Shoot, should I do a `julia -0.5-`? I really just want to exclude 0.4 here

Comment (tkelman) [inline Blink/versions/0.4.0/requires]: yeah, or be specific with one of the rc's that are already available

**Comment by spencerlyon2** [comment]
Ok trying again...

**Pull request merged**